tests/test_random_search.py

```diff
@@ -204,6 +204,50 @@ def test_rand_loguniform(plot):
     assert pred_samples.max() <= v2_max
 
 
+def test_rand_inv_loguniform(plot):
+    # Calculates that the
+
+    v2_min = 0.5
+    v2_max = 1.0
```

Comment:
add some dynamic range on this case to test that the log is working well. can we change this to 
```suggestion
    v2_max = 1e3
    v2_min = 1e-3
```